src/builders.jl

```diff
@@ -62,6 +62,42 @@ function build(builder::Short, rng, n, m)
         Flux.Dense(n_hidden, m, init=init))
 end
 
+"""
+    MLP(; hidden=(10,), σ=Flux.sigmoid, rng=GLOBAL_RNG)
+
+MLJFlux builder that constructs a Multi-layer perceptron network. The ith element of `hidden` represents the number of neurons in the ith hidden layer. An activation function `σ` is applied between each layer.
+
+The each layer is initialized using `Flux.glorot_uniform(rng)`. If
+`rng` is an integer, it is instead used as the seed for a
+`MersenneTwister`.
+
+"""
+mutable struct MLP{N} <: MLJFlux.Builder
+    hidden::NTuple{N, Int}  # count first and last layer
+    σ
+end
+MLP(; hidden=(10,), σ=Flux.sigmoid) = MLP(hidden, σ)
```

Comment:
SkLearn's `MLP{Classifier|Regressor}` use 100 hidden units and ReLU instead of sigmoid. Was there a particular reason for the choice of these hyperparams?

Reply:
No particular reason. I think I grabbed sigmoid just because I started by looking at the implementation of `Short`. ReLU will probably give a faster model for predictions.